include/cista/containers/rtree.h

```diff
@@ -781,13 +778,13 @@ struct rtree {
    * @param f The output stream to write to
    */
   void write_header_bin(std::ostream& f) {
-    f.write((char*)&rect_, sizeof rect_);
-    f.write((char*)&root_, sizeof root_);
-    f.write((char*)&free_list_, sizeof free_list_);
-    f.write((char*)&count_, sizeof count_);
-    f.write((char*)&height_, sizeof height_);
+    f.write(static_cast<char*>(static_cast<void*>(&rect_)), sizeof rect_);
```

Comment:
reinterpret_cast?

Reply:
Would work to, but I am not sure if implementation defined behavior could break this in the future

Reply:
changed it to reinterpret_cast